src/sql/base/browser/ui/colorbox/colorbox.ts

```diff
@@ -40,11 +40,10 @@ export class Colorbox extends Widget {
 		this.radioButton.setAttribute('aria-label', opts.color);
```

Comment:
It would be great to have a friendly name along with the hex value for screen readers.